payroll-and-expenses/sample-payroll.md

```diff
@@ -0,0 +1,220 @@
+# Create a sample Payroll
```

Comment:
Where did you add  the information about having to add a custom field to the employee